docs/user/docs/sessions/request.md

```diff
@@ -18,18 +18,24 @@ Please follow these steps to request a session:
    either the latest commit or complete commit history of the primary model of
    this project. Instead of choosing a single branch, it is possible to click
    the checkbox to mark that all branches should be downloaded.
-
-    !!! note ""
-        Choosing all revisions and only the latest commit clones only the latest commits for all branches.
-
-1. Click the `Request session` button.
-1. You should see the state of your request. It can take up to 2 minutes, until
+2. Click the `Request session` button.
```

Comment:
```suggestion
1. Click the `Request session` button.
```

I prefer using 1 everywhere and let Markdown calculate the numbers. Otherwise, users without automatic numbering in their IDEs have to manually update all numbers.